id: 1
start_position_x: 5
start_position_y: 2
start_position_yaw: 0